### PR TITLE
KBM, added 'chords', 'start app' and 'open uri'

### PR DESCRIPTION
**Pull request by jefflord**
Basic information on new features.

### CODE REVIEW

File: hub/powertoys/keyboard-manager.md [hunk 1]
```diff
@@ -122,6 +132,20 @@ For example, to replace the shortcut <kbd>Ctrl</kbd>+<kbd>G</kbd> with `Hello!`
...
+| :---            | :---          |
+| App  | This is the path to an executable. Environment variables will be expanded.     |
+| Args         | Arguments that will be sent to the app.   |
+| Start in     | The working directory for the app to start in.  |
+| Elevation    | Specify the elevation level to start the app. The options include Normal, Elevated, and Different User.  |
+| If running | What action should be taken when this shortcut is activated while the app is already running? The options are: Show Window, Start another instance, Do nothing, Close, End task.   |
+| Visibility   | Specifies whether the app will be visible. This is useful when the app is a console or something you don't want to see.  |
+
+### Remap a shortcut to open a URI
+This type of shortcut action will open a URI. The only option is the actual Path/URI. Almost anything you can issue on the command line should work. You can find some other examples here: https://learn.microsoft.com/windows/uwp/launch-resume/launch-app-with-uri
```
Comment (alvinashcraft) [inline hub/powertoys/keyboard-manager.md]: Adding the link inline and fixing the build suggestion for the absolute URL.
```suggestion
This type of shortcut action will open a URI. The only option is the actual Path/URI. Almost anything you can issue on the command line should work. See [Launch an app with a URI](/windows/uwp/launch-resume/launch-app-with-uri) for more examples.
```

**Comment by jaimecbernardo** [comment]
Wow @jefflord , thanks for the updated docs! 👍 🎉 
@alvinashcraft , this should be merged for the .79 release (early next week is the plan)
Thank you!

**Review by alvinashcraft — APPROVED** [review]
Thanks, @jefflord. This looks great. I'll merge next week when the release goes live.

**Review by bizcad — COMMENTED** [review]
It looks like you have a typo.  You define a chord as Shirt + Control.  Shouldn't be Shift + Control?

**Comment by jefflord** [comment]
> It looks like you have a typo. You define a chord as Shirt + Control. Shouldn't be Shift + Control?

Good eye, who can change this? @alvinashcraft, do we need a new PR?

**Comment by alvinashcraft** [comment]
It's merged so we'll need another PR. I can create one tomorrow morning when I'm back online if nobody else creates one before then. Thanks for spotting that @bizcad.

**Comment by jefflord** [comment]
> > It looks like you have a typo. You define a chord as Shirt + Control. Shouldn't be Shift + Control?
> 
> Good eye, who can change this? @alvinashcraft, do we need a new PR?

Also, the markdown failed to format as table.

image

Typo aside, (Shirt), it looks good in the editor:

image

**Comment by alvinashcraft** [comment]
Maybe it's the commas. Everything else looks the same as the other tables. I'll try escaping them. I can create the PR in our private repo so I can preview it on our review website.

**Comment by alvinashcraft** [comment]
It should look correct now. My PR is live. Thanks again!

**Comment by Jay-o-Way** [comment]
@jefflord @alvinashcraft I just saw the crooked MD table online. Who is picking that up? 
Besides that - also check the (markdown) formatting against MS guide please. E.G. "headers, tables and such, should be surrounded by one blank line" and UI elements should be in **bold**, etc.
https://learn.microsoft.com/contribute/content/markdown-reference

**Comment by alvinashcraft** [comment]
@Jay-o-Way It looks fine to me now: https://learn.microsoft.com/en-us/windows/powertoys/keyboard-manager#shortcuts-with-chords

I followed up with another PR a while ago to fix that table and added line breaks.

**Comment by Jay-o-Way** [comment]
This one ain't:
image
image

**Comment by alvinashcraft** [comment]
Ah, ok. I'll fix that today too. Thanks!

**Comment by alvinashcraft** [comment]
The page is updated but the site is taking a while to update. I can see the line breaks added in the source file in the main branch: https://raw.githubusercontent.com/MicrosoftDocs/windows-dev-docs/docs/hub/powertoys/keyboard-manager.md

It should appear on the live site shortly. If not, I'll see if I can push another minor change to force it to deploy again.

**Comment by Jay-o-Way** [comment]
Can confirm the table looks good now. The formatting, oh well, #details...

**Pull request merged**